packages/appstore/src/components/ready-to-update-wallets/ready-to-update-wallets.tsx

```diff
@@ -0,0 +1,110 @@
+// TODO: Once everything is merged place this in main component at the last step
+import React from 'react';
+import {
+    Button,
+    Checkbox,
+    Modal,
+    DesktopWrapper,
+    MobileWrapper,
+    MobileDialog,
+    Div100vhContainer,
+} from '@deriv/components';
+import { localize } from '@deriv/translations';
+import ReadyToUpgradeForm from './ready-to-upgrade-form';
+import { useStore } from '@deriv/stores';
+import { observer } from 'mobx-react-lite';
```

Comment:
use observer from @deriv/stores, not from mobx-react-lite

Reply:
Done 😃 👍🏼